engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/NotEmptyValidatorEvaluator.scala

```diff
@@ -0,0 +1,16 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import java.lang.reflect.Parameter
+
+import javax.annotation.Nullable
+import pl.touk.nussknacker.engine.api.definition.{MandatoryValueValidator, ParameterValidator}
+
+object NotEmptyValidatorEvaluator extends ValidatorEvaluator {
+
+  override def evaluate(param: Parameter): Option[ParameterValidator] = {
+    param.getAnnotation(classOf[Nullable]) match {
+      case _: Nullable => None
+      case _ => Some(MandatoryValueValidator)
```

Comment:
IMO we should hande here also `Option[]` and `Optional[]` - let's keep all of them in one place.

Reply:
Done